python_modules/dagster/dagster/_core/execution/plan/step.py

```diff
@@ -161,7 +160,7 @@ def __new__(
                 so.name: so
                 for so in check.sequence_param(step_outputs, "step_outputs", of_type=StepOutput)
             },
-            tags=normalize_tags(tags, warn_on_deprecated_tags=False),
```

Comment:
This is only ever constructed from `ExecutionStepSnapshot` (which already has str/str tags) or tags from `Node` which have already been normalized.